packages/product-react/src/components/Common/FieldWrapper.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+// This is only needed when we have a prepended icon for wrapping the text field
+const FieldWrapper: React.FC<{
+  renderWrapper?: boolean;
+  fieldClass?: string;
+}> = ({ children, renderWrapper, fieldClass }) =>
+  renderWrapper ? (
+    <div
+      className={fieldClass}
+      // className="ray-select-field__wrapper"
+    >
+      {children}
+    </div>
+  ) : (
+    <>{children}</>
```

Comment:
it might read a little clearer if you had this:

```
if (!renderWrapper` {
   return children;
}
```

Reply:
if you copy/pasted what was already there feel free to ignore this

Reply:
ya this was already there